File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/magic3/p8.c
#include<stdio.h>
void main(){

	int arr[6]={1,2,3,4,5};
	++arr[1];
	arr[arr[1]]++;

	arr[arr[arr[1]]]++;

	for(int i=0;i<6;i++){

		printf("%d\n",arr[i]);
	}
}

File: java-main/java-main/DSA/Array/p8.java
import java.util.Arrays;

class ArrayRotation {
    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 4, 5};
        int rotationSteps = 3; 

        System.out.println("Original Array: " + Arrays.toString(arr));
        
        int[] rotatedArr = new int[arr.length];
        
        for (int i = 0; i < arr.length; i++) {
            int newPosition = (i + rotationSteps) % arr.length; 
	    rotatedArr[newPosition] = arr[i]; 
        }

        System.out.println("Rotated Array: " + Arrays.toString(rotatedArr));
    }
}

